staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_secret.go

```diff
@@ -62,7 +62,7 @@ func (c *FakeSecrets) List(opts v1.ListOptions) (result *core_v1.SecretList, err
 	if label == nil {
 		label = labels.Everything()
 	}
-	list := &core_v1.SecretList{}
+	list := &core_v1.SecretList{ListMeta: obj.(*core_v1.SecretList).ListMeta}
```

Comment:
I moved that into: https://github.com/kubernetes/kubernetes/pull/63800 (will rebase once that one is merged).